checklist/src/main/scala/checklist/Checked.scala

```diff
@@ -0,0 +1,138 @@
+package checklist
+
+import cats.implicits._
+import cats.{Semigroup, Monad, Monoid}
+import cats.data.{NonEmptyList, Ior, Validated}
+
+sealed trait Checked[E, W, R] {
```

Comment:
Make R covariant, maybe.

Reply:
My gut feeling is that all three should be covariant.

Reply:
Probably, yeah. No reason they couldn't all be.